setup_cfg_fmt.py

```diff
@@ -292,15 +285,12 @@ def _py_classifiers(
         if minimum not in exclude:
             versions.add(minimum)
             versions.add(minimum[:1])
-        if minimum == (2, 7):
-            minimum = (3, 0)
-        else:
-            minimum = (minimum[0], minimum[1] + 1)
+        minimum = (minimum[0], minimum[1] + 1)
 
     classifiers = [
         f'Programming Language :: Python :: {_v(v)}' for v in versions
     ]
-    if (3,) in versions and (2,) not in versions:
+    if (3,) in versions:
```

Comment:
is this ever false now?

Reply:
Only in this test: https://github.com/mxr/setup-cfg-fmt/blob/555ab889d6bbb1b950df32479e527ffde1038823/tests/setup_cfg_fmt_test.py#L1124-L1134

Should I remove it?

Reply:
oh interesting -- yeah let's just always set it 🤷

Reply:
Sure